lib/Runtime/Library/WebAssemblyInstance.cpp

```diff
@@ -322,7 +322,7 @@ void WebAssemblyInstance::LoadImports(
             {
                 Assert(env->GetWasmFunction(counter) == nullptr);
                 AsmJsScriptFunction* func = AsmJsScriptFunction::FromVar(prop);
-                if (!wasmModule->GetSignature(counter)->IsEquivalent(func->GetSignature()))
+                if (!wasmModule->GetFunctionSignature(counter)->IsEquivalent(func->GetSignature()))
```

Comment:
I agree, it was useful before we had wasm function thunks for imported functions